File: mParticle-Apple-SDK/Identity/MPAudience.m
//
//  MPAudience.m
//  mParticle-Apple-SDK
//
//  Created by Brandon Stalnaker on 2/20/24.
//

#import "MPAudience.h"

// Internal keys
NSString * const kMPAudienceMembershipKey = @"audience_memberships";
NSString * const kMPAudienceIdKey = @"audience_id";

@implementation MPAudience

- (instancetype)initWithAudienceId:(NSNumber *)audienceId {
    self = [super init];
    if (self) {
        _audienceId = audienceId;
    }
    return self;
}

@end

File: mParticle-Apple-SDK/Identity/MPAudience.swift
//
//  MPAudience.swift
//  mParticle-Apple-SDK
//
//  Created by Denis Chilik on 10/28/25.
//

@objcMembers
class MPAudience: NSObject {
    static let kMPAudienceMembershipKey = "audience_memberships"
    static let kMPAudienceIdKey = "audience_id"
    
    public let audienceId: NSNumber
    
    public init(audienceId: NSNumber) {
        self.audienceId = audienceId
    }
}
